SYCL/InvokeSimd/spec/IMPL_SUBGROUP/popcnt.cpp

```diff
@@ -0,0 +1,19 @@
+// TODO: enable on Windows once driver is ready
```

Comment:
I think this one is not part of the spec, it should rather go to the 4th category we discussed

Reply:
@kbobrovs,
As we discussed there will be 4 categories: Spec, Feature, Regression and Bench/Workload (last one goes to closed-source repo). Do I need to move only [IMPL_SUBGROUP/popcnt.cpp](https://github.com/intel/llvm-test-suite/pull/1360/files/82e98752b53518b18debc62b79a7f2199dc3e85b#diff-606732d027b58bb5a8c26d714b1b59290e70081bf371e40b873ec83bbe24d0b4) test to other category or all tests in IMPL_SUBGROUP directory? And in what category exactly?

Reply:
I think the following should not be part of the SPEC subsuite:
- *pvc*
- popcnt, scale

Reply:
The main reason why there is a separate test for PVC is because the permissible simd sizes are different for this platform and I don't know how to recognize the platform at compile time.

Moving PVC version of the test to another category will lead to code duplication or using relative path to common header. Should I move all simd_size tests to the 'regression' category?

Reply:
I reorganized and renamed simd_size tests so there is no explicit reference to specific platforms in the names